README.md

```diff
@@ -108,9 +108,7 @@ kubectl -n stackrox port-forward deploy/central 8000:8443
 
 2.Open <https://localhost:8000> in your browser.
 
-3.Accept certificate warnings and proceed.
```

Comment:
I disagree about removing this, I think we should say something about self-signed certs.

Reply:
How about we rephrase it? Something along the lines of:
"In a quick setup, you will encounter a self-signed certificate here, which you must accept to continue. For a custom certificate setup, consult https://docs.openshift.com/acs/configuration/add-custom-certificates.html"?
What do you think?

Reply:
Went with the suggestion of Michael in the end, I think this should be good